Mesh_3/include/CGAL/Polyhedral_complex_mesh_domain_3.h

```diff
@@ -35,7 +35,7 @@
 #include <CGAL/Polygon_mesh_processing/compute_normal.h>
 
 #include <CGAL/Mesh_3/Polyline_with_context.h>
-#include <CGAL/Polygon_mesh_processing/Detect_features_in_polyhedra.h>
+#include <CGAL/Polygon_mesh_processing/Detect_features_in_polygon_mesh.h>
```

Comment:
+1
Also, the only files starting with a capital letter in this package are the ones providing a class